README.md

```diff
@@ -31,6 +31,17 @@ If you are the current maintainer of the package then create a new request using
 * The issue will get labelled as assigned, awaiting approval
 * When the package gets approved by the moderation process, close the issue (or ask for it to be closed).
 
+When you pick up a request for either a new package or a new maintainer, remember to keep the maintainers of this repository updated on the current status of the work being done.
+A comment from the one picking up a request is expected to be done at the following stages of a request:
```

Comment:
Another blank line needed her?

Reply:
1 blank line should be enough between a paragraph and a list.
Or do you mean separating this line into its own paragraph?

Reply:
Yeah. It looked like you were putting that into a separate paragraph. If not, resolve this comment.